guides/main_concepts/documents.md

```diff
@@ -1,22 +1,26 @@
 # Documents
 
-**Documents** are objects composed of fields containing any type of data.</br>
-A **field** is composed of an **attribute** and its associated data.
+**Documents** are objects composed of fields which can store any type of data.</br>
+Each **field** contains an **attribute** and its associated data.
 
 ![document structure](/document_structure.svg)
 
 #### Wording
 
+- **Attribute**: An attribute is like a key (`"id"`, `"title"`, `"description"` and `"type"`).
+- **Field**: A field, or a key-value pair, is a set of two data items linked together. Here, an attribute and its corresponding data (i.e, `"title": "Interstellar"`).
+- # **Document**: A document is an object which contains a list of fields in curly brackets.
 - **Attribute**: An attribute is the name of a field, like a key.
 - **[Field](/guides/main_concepts/documents.md#fields)**: A field, or a key-value pair, is a set of two data items linked together: an attribute and its corresponding data
 - **Document**: A document is an object which collects a list of fields between curly brackets.
 - **[Primary key](/guides/main_concepts/documents.md#primary-key)**: The attribute of the field which contains the unique identifier of the documents.
 - **[Document id](/guides/main_concepts/documents.md#document-id)**: The unique value of a document.
+  > > > > > > > cf20813fb99994943b619a3e29426e08e8bd2c71
 
 ## Structure
 
-A document is added to MeiliSearch in `JSON format`.<br/>
-When using the [route to add new documents](/references/documents.md#add-or-update-documents), all documents should be sent in an array. And this, even if there is only one document.
+Documents are stored in `JSON format`.<br/>
```

Comment:
This changes the sentence to an incorrect information. 
Documents are sent to MeiliSearch using the JSON format. Once it is in meilisearch it is not stored in JSON format.
```suggestion
Documents are sent to MeiliSearch in `JSON format`.<br/>
```

Reply:
Thanks for the correction!